### ISSUE TITLE
Upgrade url-parse dep

### ISSUE THREAD

**Issue by dandv** [sockjs/sockjs-client#447]
Vulnerability highlighted by npm's security audit:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Open Redirect                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ url-parse                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ preact-cli [dev]                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ preact-cli > webpack-dev-server > sockjs-client >            │
│               │ eventsource > original > url-parse                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/678                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by 3rd-Eden** [comment]
Shouldn't it just install latest anyways as it's added using a semver range `"url-parse": "^1.1.8"` - As they using `^` instead of `~`, it should include minor versions as well so it will catch 1.4.3 of URL-parse  that includes the fix.

If you look at the path, it's actually the `original` dependency of `eventsource` that is causing the issue. But you're in luck, as I'm the author of `original` and `url-parse`, so I'll make sure that the `original` module gets `patch` bumped with the latest `url-parse`

**Comment by 3rd-Eden** [comment]
> + sockjs-client@1.1.5
> added 11 packages from 14 contributors and audited 23 packages in 1.507s
> found 0 vulnerabilities

Published `original@1.0.2` which generates the output above.

**Comment by dandv** [comment]
Thanks for updating `original`! I saw this vulnerability in this path as well:

    preact-cli > webpack-dev-server > sockjs-client > url-parse
